lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -314,15 +309,9 @@ function M.get()
 
 	-- lazy.nvim updates
 	components.active[1][14] = {
-		provider = function()
-			if has_lazy() then
-				return require("lazy.status").updates()
-			else
-				return " "
-			end
-		end,
+		provider = function() return require("lazy.status").updates() end,
```

Comment:
I think here a function is not necessarily, if we don't check for any condition.

And then it looks like this: `provider = return require("lazy.status").updates` without the parenthesis. But I will check.

Reply:
If the requiring isn't wrapped by a function, it will be called once the file is loaded no matter the `enabled` returns true or false, which means it might cause an error if the user doesn't use lazy.nvim

Reply:
I understand. Than it's better to be a function. 👍🏻